src/python/pants/backend/java/compile/javac_test.py

```diff
@@ -91,6 +98,43 @@ def rule_runner() -> RuleRunner:
 )
 
 
+@dataclass(frozen=True)
+class RenderedClasspath:
+    """The contents of a classpath, organized as a key per entry with its contained classfiles."""
```

Comment:
Is the key a filename of the jar?

Reply:
Yea, correct. This is test-only, so I didn't apply quite as much rigor.